Formula/jvm-mon.rb

```diff
@@ -22,9 +22,10 @@ def install
     rm_f Dir["bin/*.bat"]
     libexec.install Dir["*"]
 
-    (bin/"jvm-mon").write_env_script "#{libexec}/bin/jvm-mon",
-      Language::Java.java_home_env("1.8")
-    system "unzip", "-j", libexec/"lib/j2v8_macosx_x86_64-4.6.0.jar", "libj2v8_macosx_x86_64.dylib", "-d", libexec
+    (bin/"jvm-mon").write_env_script libexec/"bin/jvm-mon", Language::Java.java_home_env("1.8")
+    os = OS.mac? ? "macosx" : "linux"
+    jar = libexec/"lib/j2v8_#{os}_x86_64-4.6.0.jar"
+    system "unzip", "-j", jar, shared_library("libj2v8_#{os}_x86_64"), "-d", libexec
```

Comment:
Maybe we can postpone the `unzip` to `post_install` to preserve the `:all` bottle. Unless `unzip` takes an unreasonable amount of time?

Reply:
Actually, I think `jvm-mon` automatically unpacks shared library at runtime if it is not available.

Linux is failing due to `_JAVA_OPTIONS` we set, specifically the `-Duser.home`.

Reply:
Looks like we lost `all` bottle anyway as there are `/usr/local` scattered throughout source for node modules `blessed`, `dashdash`, and `picture-tube`.